src/app/api/vehicle/model/letters.ts

```diff
@@ -1,28 +0,0 @@
-/**
```

Comment:
Aren't these generated? Have you changed the swagger?

Reply:
I created the letter components manually, tried basing it off the plates components as closely as possible - including the api models, so if it's not needed then that's fine.